File: contracts-java/src/main/java/com/template/contracts/AccountsDealContract.java
package com.template.contracts;

import net.corda.core.contracts.CommandData;
import net.corda.core.contracts.Contract;
import net.corda.core.transactions.LedgerTransaction;
import org.jetbrains.annotations.NotNull;

public class AccountsDealContract implements Contract {
    // Used to identify our contract when building a transaction.
    public static String ID = "com.template.contracts.AccountsDealContract";

    @Override
    public void verify(@NotNull LedgerTransaction tx) throws IllegalArgumentException {

    }

    public interface Commands extends CommandData {
        class CreateDeal implements Commands {}
    }
}

File: contracts/src/main/kotlin/com/template/contracts/AccountsDealContract.kt
package com.template.contracts

import net.corda.core.contracts.CommandData
import net.corda.core.contracts.Contract
import net.corda.core.transactions.LedgerTransaction


class AccountsDealContract: Contract {

    companion object {
        // Used to identify our contract when building a transaction.
        const val ID = "com.template.contracts.AccountsDealContract"
    }

    interface Commands : CommandData {
        class CreateDeal : Commands
    }


    override fun verify(tx: LedgerTransaction) {

    }


}